server/appengine/src/main/java/who/WhoServletModule.java

```diff
@@ -33,8 +33,6 @@ protected void configureServlets() {
     serve("/remote_api").with(new RemoteApiServlet());
 
     serve("/app").with(new AppStoreServlet());
-    serve("/terms").with(forwardTo("/terms.pdf"));
```

Comment:
Old versions of the app will still point to these URLs.  Do you want to HTTP 301 them to the new urls?

Reply:
If the app was already released then we'd have to do this. Given the small number of users, that they've already been provided with a working link at the time they accepted the TOS and PN.... and that we'll be pushing a new build first thing next week, I'd rather focus the limited resources on more pressing issues.

Reply:
Advay - the other thought is that as soon as you push the Static Content Service, the old apps should pick up the new location and show that in the UI, so they shouldn't ever need a redirect. That'll make for a nice test in a way.

Reply:
I'm not sure what the Static Content Service has to do with this though.  That is for the YAMLs, not for the onboarding/ToS screens which are coded.

Reply:
Fair point but I'll hold to the original logic. What I'll probably do is push the new binary, then wait a day for people to upgrade and then switch out the server.